ble/DiscoveredCharacteristic.h

```diff
@@ -137,13 +137,13 @@ class DiscoveredCharacteristic {
      */
     ble_error_t write(uint16_t length, const uint8_t *value) const;
 
-    /** 
+    /**
      * Same as above but register the callback wich will be called once the data has been written
      */
     ble_error_t write(uint16_t length, const uint8_t *value, const GattClient::WriteCallback_t& onRead) const;
 
-    void setupLongUUID(UUID::LongUUIDBytes_t longUUID) {
-        uuid.setupLong(longUUID);
+    void setupLongUUID(UUID::LongUUIDBytes_t longUUID, UUID::BitOrder_t order = UUID::MSB) {
```

Comment:
side note: at some point, maybe it would be nice if we were able to set any kind of UUID, not just the long UUIDs. 
What do you think ? I can open a ticket for this.